api/server/database/gorm/odata.go

```diff
@@ -848,19 +848,15 @@ var schemaMetas = map[string]odatasql.SchemaMeta{
 	"Finding": {
```

Comment:
```suggestion
	findingSchemaName: {
```
can also be used for relationships against "Finding" table